core/trino-main/src/test/java/io/trino/sql/planner/TestInsert.java

```diff
@@ -187,10 +188,42 @@ public void testCreateTableAsSelectUnpartitioned()
                                         values("column1", "column2")))));
```

Comment:
`testCreateTableAsSelectUnpartitioned` should use `withPreferredPartitioning`. Please extract as a separate commit